app/shared/components/Tools/Form/CreateAccount.js

```diff
@@ -0,0 +1,420 @@
+    }
+
+    const decimalBalance = Decimal(EOSbalance);
+    const decimalDelegatedBw = Decimal(delegatedBw.split(' ')[0]);
+    const decimalDelegatedCpu = Decimal(delegatedCpu.split(' ')[0]);
+
+    const decimalDelegatedResources = decimalDelegatedBw.plus(decimalDelegatedCpu);
+
+    if (ramPrice.plus(decimalDelegatedResources).greaterThan(decimalBalance)) {
+      if (delegatedBw > 1) {
+        formErrors.delegatedBw = 'not_enough_balance';
+      } else if (delegatedCpu > 1) {
+        formErrors.delegatedCpu = 'not_enough_balance';
+      } else {
+        formErrors.ramAmount = 'not_enough_balance';
+      }
+
+      submitDisabled = true;
+    }
+
+    return { formErrors, submitDisabled };
+  }
+
+  onBack = () => {
+    this.setState({
+      confirming: false
+    });
+  }
+
+  onConfirm = () => {
+    this.setState({
+      confirming: false
+    });
+
+    const {
+      actions
+    } = this.props;
+
+    const {
+      createAccount
+    } = actions;
+
+    const {
+      accountName,
+      activeKey,
+      delegatedBw,
+      delegatedCpu,
+      ownerKey,
+      ramAmount
+    } = this.state;
+
+    createAccount(
+      accountName,
+      activeKey,
+      delegatedBw,
+      delegatedCpu,
+      ownerKey,
+      ramAmount
+    );
+  }
+
+  render() {
+    const {
+      onClose,
+      system,
+      t
+    } = this.props;
+
+    const {
+      accountName,
+      activeKey,
+      delegatedBw,
+      delegatedCpu,
+      ownerKey,
+      ramAmount,
+      ramPrice
+    } = this.state;
+
+    let {
+      formErrors,
+      submitDisabled
+    } = this.state;
+
+    const shouldShowConfirm = this.state.confirming;
+    const shouldShowForm = !shouldShowConfirm;
+
+    if (accountName &&
+        accountName.length === 12 &&
+        system.ACCOUNT_AVAILABLE === 'FAILURE' &&
+        system.ACCOUNT_AVAILABLE_LAST_ACCOUNT === accountName) {
+      formErrors.accountName = 'account_name_not_available';
+      submitDisabled = true;
+    }
+
+    if (system.ACCOUNT_AVAILABLE === 'FAILURE') {
+      submitDisabled = true;
+    }
+
+    const formErrorKeys = Object.keys(formErrors);
+
+    const decimalDelegatedBw = delegatedBw && Decimal(delegatedBw.split(' ')[0]);
+    const decimalDelegatedCpu = delegatedCpu && Decimal(delegatedCpu.split(' ')[0]);
+
+    let totalCost;
+
+    if (decimalDelegatedBw && decimalDelegatedCpu && ramAmount) {
+      totalCost = decimalDelegatedBw.plus(decimalDelegatedCpu).plus(ramPrice);
+    }
+
+    const shouldShowAccountNameWarning = accountName && accountName.length !== 12
+
+    const shouldShowDelegatedResourceWarning =
+      (decimalDelegatedBw &&
+        decimalDelegatedBw.lessThan(1) &&
+        decimalDelegatedBw.greaterThan(0)) ||
+      (decimalDelegatedCpu &&
+        decimalDelegatedCpu.lessThan(1) &&
+        decimalDelegatedCpu.greaterThan(0));
+
+    return (
+      <Segment
+        loading={system.CREATEACCOUNT === 'PENDING'}
+        style={{ minHeight: '100px' }}
```

Comment:
Why is this here? I don't think we need any minimum heights do we?

Reply:
Nice catch! Copied that over from the staking form where it use to be required before we refactored staking to use your `GlobalTransactionHandler` component. I'll remove it here and from the staking form.